Stylesheets/Components/avatar.css

```diff
@@ -0,0 +1,46 @@
+.avatar-circle{
+    max-width: 100%;
+    height: auto;
+    border-radius: 50%;
+    height: 2.5rem;
+    width: 2.5rem;
+}
+.avatar-square{
+    max-width: 100%;
+    height: auto;
+    border-radius: 10%;
+    height: 2.5rem;
+    width: 2.5rem;
+}
+.avatar img{
+    height: 2.5rem;
+    width: 2.5rem;
+}
+
+.avatar-shadow{
+    box-shadow: 10px 10px 2px 10px #888888;
+}
+
+.sm{
+    width: 1.5rem;
+    height: 1.5rem;
+    
+}
+
+.md{
+    width: 2.5rem;
+    height: 2.5rem;
+    
+}
+
+.lg{
+    width: 3.5rem;
+    height: 3.5rem;
+    
+}
+
```

Comment:
the '.sm', '.md' & '.lg' classes here, seem like general classes which could be applied to any element, but actually, they are specific to avatars only, so please change their names accordingly, like `.avatar-sm, .avatar-md etc.`

Reply:
Hi Krituraj,
Thanks for pointing it out, I have changed the class names as per your suggestion